website/pages/docs/k8s/connect/overview.mdx

```diff
@@ -275,6 +275,15 @@ Annotations can be used to configure the injection behavior.
     consul.hashicorp.com/service-meta-bar: baz
   ```
 
+- `consul.hashicorp.com/sidecar-proxy-` - override default resource settings for
+  the Envoy sidecar proxy container.
```

Comment:
🤔 should we omit saying Envoy here in case we support other proxies? I think it'll be clear with just "sidecar proxy container".